### PR TITLE
adding Querystring parser

### PR DESCRIPTION
**Pull request by jaroslawk**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jaroslawk** [comment]
in contact with author.

File: recipes/querystring_parser/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+test:
+  imports:
+    - querystring_parser
+
+about:
+  home: https://github.com/bernii/querystring-parser
+  license: MIT
+  license_family: MIT
+  license_file:
```
Comment (chrisburr) [inline recipes/querystring_parser/meta.yaml]:
```suggestion
  license_file: LICENSE
```

File: recipes/querystring_parser/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - querystring_parser
+
+about:
+  home: https://github.com/bernii/querystring-parser
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: QueryString parser for Python/Django that correctly handles nested dictionaries
+  doc_url: 
+  dev_url:
```
Comment (chrisburr) [inline recipes/querystring_parser/meta.yaml]:
```suggestion
  dev_url: https://github.com/bernii/querystring-parser
```

File: recipes/querystring_parser/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  imports:
+    - querystring_parser
+
+about:
+  home: https://github.com/bernii/querystring-parser
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: QueryString parser for Python/Django that correctly handles nested dictionaries
+  doc_url:
```
Comment (chrisburr) [inline recipes/querystring_parser/meta.yaml]: If there isn't a URL these fields shouldn't be specified:
```suggestion
```

File: recipes/querystring_parser/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  skip: True  # [not win]
+
+requirements:
+  host:
+    - pip
+    - python
+    - requests
+  run:
+    - python
+    - requests
```
Comment (chrisburr) [inline recipes/querystring_parser/meta.yaml]:
```suggestion
    - requests
    - six
```

File: recipes/querystring_parser/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [not win]
+
+requirements:
+  host:
+    - pip
+    - python
+    - requests
```
Comment (chrisburr) [inline recipes/querystring_parser/meta.yaml]: By the looks of it this isn't needed to run the `setup.py`
```suggestion
```

File: recipes/querystring_parser/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4c31b547c77b927a8aceccd6fa37f8152aa92fe5654a43d7363fcee8cf6d8aea
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [not win]
```
Comment (chrisburr) [inline recipes/querystring_parser/meta.yaml]: Could this be a noarch package instead:
```suggestion
  noarch: python
```

File: recipes/querystring_parser/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+  license_family: MIT
+  license_file: 
+  summary: QueryString parser for Python/Django that correctly handles nested dictionaries
+  doc_url: 
+  dev_url: 
+
+extra:
+  recipe-maintainers:
+    - jaroslawk
+    - paulmelnikow
```
Comment (chrisburr) [inline recipes/querystring_parser/meta.yaml]: https://github.com/bernii/querystring-parser/issues/30#issuecomment-480328020

Comment (jaroslawk) [reply recipes/querystring_parser/meta.yaml]: @paulmelnikow fyi

**Pull request merged**